ci/azure-pipelines-template.yml

```diff
@@ -47,14 +47,37 @@ jobs:
     displayName: Clippy
     condition: ne( variables['rustup_toolchain'], 'nightly' )
 
-  - ${{ if ne(parameters.name, 'Windows') }}:
+  - ${{ if eq(parameters.name, 'Linux') }}:
+    - script: |
+        cargo run --bin pyoxidizer -- init --pip-install cryptography ~/pyapp
+        cat ci/pyapp.py | cargo run --bin pyoxidizer -- run ~/pyapp
+      displayName: Build Oxidized Application
+
+  - ${{ if eq(parameters.name, 'macOS') }}:
+    - script: brew install openssl@1.1
```

Comment:
this is unnecessary, as it is pre-installed, but it is only a few seconds.  Happy to remove it though.